build.sh

```diff
@@ -77,12 +77,18 @@ cd $BUILDDIR/$ARCH
 
 # =========== libicuXX.so ===========
 
-[ -e libicuuc.so ] || {
+[ -e libicuuc$LIBSUFFIX.so ] || {
 
 	[ -e ../icu4c-55_1-src.tgz ] || exit 1
 
 	tar xvf ../icu4c-55_1-src.tgz
 
+	# The ENVVAR LIBSUFFIX should add the suffix only to the libname and not to the symbols.
+	# ToDo: Find the right way in Swift to refer to an alternative library with symbol prefixing or any other method to remove this.
+	if $[ -n "$LIBSUFFIX" ]; then
```

Comment:
@gonzalolarralde is the `$` in front of the `[]` supposed to be here? This line is failing on my machine